### PR TITLE
added two jobs

### PR DESCRIPTION
**Pull request by nairihar**

### CODE REVIEW

**Review by arrustamyan — CHANGES_REQUESTED** [review]
There are couple of issues with this PR:

- Both jobs still depend on the same MINIMUM_JOB_TIMEOUT for iteration frequency which is not efficient as employees should be fetched far more rarely
- There is still no way to define dependency between jobs (fetch transactions after employees have been sent) which means this works the same way as before

File: api/src/controllers/employeesController.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ EmployeeController.addEmployees = async (req, res, next) => {
         });
 
         await Promise.all(upsertPromises);
-        res.end('Employees are successfully inserted!');
+        res.status(201).end('Employees are successfully inserted!');
```
Comment (arrustamyan) [inline api/src/controllers/employeesController.js:28]: good one!

File: docker-compose.yml [hunk 2]
```diff
@@ -36,13 +38,17 @@ services:
...
+        - postgres
       volumes:
         - ./scheduler/:/src
       environment:
         - API_ENDPOINT=http://api:8080/api
         - NODE_ENV=development
         - LOG_LEVEL=info
-        - REQUEST_INTERVAL=100000
+        - REQUEST_EMPLOYEES_INTERVAL=100000
+        - REQUEST_TRANSACTIONS_INTERVAL=86400000
```
Comment (arrustamyan) [inline docker-compose.yml]: why is transactions interval bigger than employees interval?

**Review by arrustamyan — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (nairihar) [reply docker-compose.yml]: confused in places

**Review by arrustamyan — APPROVED** [review]

**Pull request merged**